src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java

```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- *
+ * <p>
```

Comment:
Delete this, and don't forget to add author :)

Reply:
Oh no